src/containers/TopicArticlePage/components/TopicArticleContent.jsx

```diff
@@ -11,12 +11,8 @@ import PropTypes from 'prop-types';
 import { injectT } from '@ndla/i18n';
 import BEMHelper from 'react-bem-helper';
 import headingPlugin from '../../../components/SlateEditor/plugins/heading';
-import {
-  TextField,
-  PlainTextField,
-  RemainingCharacters,
-  classes as fieldClasses,
-} from '../../../components/Fields';
+import { TextField } from '../../../components/Fields';
+import LearningResourceIngress from '../../LearningResourcePage/components/LearningResourceIngress';
```

Comment:
Maybe rename to IngressField (or something similar) and move to main component folder?